rodas2021rehamiltonian/openreview/packages.tex

```diff
@@ -0,0 +1,15 @@
+\usepackage[]{neurips_2019}
```

Comment:
Please remove this import - we no longer need neurips style files as we will be using ReScience file.

Reply:
Fixed in commit dcea8addb4e018c8b16534d83a57b3a851929ada